src/worker/token.worker.ts

```diff
@@ -31,7 +50,9 @@ const messageHandler = async ({
   const { audience, scope } = auth || {};
 
   try {
-    const body = JSON.parse(fetchOptions.body);
+    const body = useFormData
+      ? (parseFormData(fetchOptions.body) as any)
```

Comment:
Can't we leave out the cast to any here ? 
![image](https://user-images.githubusercontent.com/2146903/127466074-78cf28dd-5de7-4071-be07-bc1140a18cc2.png)

Reply:
Yeah! Works for me; `parseFormData` implicitly returns `any` anyway, so I've removed this cast in 6740c33